Highrachy/ballers-api
# #171808802 Configure coveralls for ballers

#### What does this PR do?
This PR configures coveralls for ballers

#### Description of Task to be completed?
Coveralls ensure that we deliver better code. It ensures that we can see the part of the codebase not covered by the test. The current setup is set to reject codes with less than 80% coverage.

#### How should this be manually tested?
* Install the new packages with `npm install`
 * Run `npm test` on your local machine (This should run the test and show the coverage).
